cli/tests/test_next_prev_commands.rs

```diff
@@ -567,6 +567,127 @@ fn test_next_editing() {
     "###);
 }
 
+#[test]
+fn test_prev_conflict() {
+    // Make the third commit our new parent.
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+    let file_path = repo_path.join("content.txt");
+    test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "first"]);
```

Comment:
This commit doesn't seem to add any value to the test, so delete it?

Reply:
removed.

Reply:
This also doesn't seem done. Same in all of the tests, I think

Reply:
actually removed the useless first commit in all tests.